Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+RUN apt-get install -y software-properties-common wget
+
+# System deps
+RUN apt-get update
+RUN apt-get install -y build-essential \
+    sudo \
+    subversion \
+    make \
+    cmake \
+    ninja-build \
+    git \
+    clang-6.0 \
+    curl \
+    autoconf \
+    automake \
+    libtool \
+    unzip \
+    python3-dev \
+    python3-pip 
```

Comment:
Is `subversion` and `curl` necessary? See also my other comment about clang: it might be better to get rid of the host clang dependency.

Also, it would be nice to write this list in alphabetical order.

Reply:
Yep, both good points, I'd put them there for my convenience but not necessary